pkg/disk_ops.go

```diff
@@ -396,12 +397,41 @@ func (r *diskOps) RemoveDMDevice(dmDevice string) error {
 	return err
 }
 
+// sortDMDevicesForDeletion makes sure that DM devices are ordered correctly for deletion.
```

Comment:
Why did you delete the entire useful paragraph about how they must be deleted in order?

Reply:
I think the request was to be more terse in the description "present simple kind of phrasing"